recipes/Java/JodaTimeToJavaTime/DateTimeFormatter/Methods/Datetimeformatter-noequivalent.yaml

```
@@ -0,0 +1,43 @@
+id: scw:java.time:Joda-Time:datetimeformatter-noequivalent
+version: 10
+metadata:
+  name: DateTimeFormatter has no equivalent method in java.time
+  shortDescription: DateTimeFormatter has no equivalent method in java.time
+  level: marked_information
+  language: java
+  enabled: true
+  comment: |-
+    There are some methods from org.joda.time.format.DateTimeFormatter that there is no equivalent mehtod in java.time.
```

```suggestion
    There are some methods from org.joda.time.format.DateTimeFormatter that have no equivalent in java.time.
```

or at least fix the spelling mistake in the word "mehtod": "method"

---

Thanks